### PR TITLE
Desktop nav: have subroutes of Profile show up as selecting the People tab

### PR DESCRIPTION
**Pull request by cjb**
@keybase/react-hackers 

We added a People tab that's an alias to the Profile tab, but the nav selection doesn't stay on it -- profile tab actions call navigateTo([profileTab]) and the focus goes away.  So we need a People tab that is actually a Profile tab, and that's what this PR does.

### CODE REVIEW

File: shared/app/tab-bar/index.render.desktop.js [hunk 1]
```diff
@@ -45,17 +57,17 @@ const TabBar = ({onTabClick, selectedTab, username, badgeNumbers}: Props) => (
...
       />
     ))}
     <Box style={{flex: 1}} />
     <TabBarButton
       label={username}
       isNav={true}
-      selected={selectedTab === Tabs.profileTab}
-      onClick={() => onTabClick(Tabs.profileTab)}
-      badgeNumber={badgeNumbers[Tabs.profileTab]}
+      selected={selectedTab === Tabs.peopleTab}
```
Comment (chrisnojima) [inline shared/app/tab-bar/index.render.desktop.js]: I believe this is never supposed to be selected now

File: shared/app/tab-bar/index.render.desktop.js [hunk 2]
```diff
@@ -22,18 +23,29 @@ const _labels = {
...
+  ...(flags.tabPeopleEnabled ? [Tabs.profileTab] : []),
   Tabs.folderTab,
   Tabs.chatTab,
   Tabs.devicesTab,
   Tabs.settingsTab,
 ].filter(Boolean)
 
+const iconForTab = (tab, selectedTab) => {
+  const icon = tab === Tabs.profileTab
+    ? _icons[Tabs.peopleTab][selectedTab === tab ? 'selected' : 'unselected']
```
Comment (chrisnojima) [inline shared/app/tab-bar/index.render.desktop.js]: Why is this necessary?

**Comment by cjb** [comment]
@chrisnojima Thanks.  How about this change instead?

**Comment by keybase-ci-visdiff** [comment]
The commits d1aa060c711d...1c878706f4c1 introduce visual changes on linux. <details><summary>:mag_right: 1 changed</summary>

 * Changed: **App_Tab_Bar-Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1500074076871-d1aa060c711d-1c878706f4c1-linux/App_Tab_Bar-Normal-d1aa060c711d.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1500074076871-d1aa060c711d-1c878706f4c1-linux/App_Tab_Bar-Normal-1c878706f4c1.png)  
   App_Tab_Bar-Normal comparison
</summary>
